go.mod

```diff
@@ -8,6 +8,9 @@ go 1.23.2
 
 require (
 	cloud.google.com/go/storage v1.30.1
+	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.9.0
+	github.com/Azure/azure-sdk-for-go/sdk/azidentity v1.4.0
+	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/compute/armcompute v1.0.0
```

Comment:
Can we use a newer version for these modules? then it can reduce the bump in the future.

Reply:
fixed